docs/Contributing-back.rst

```diff
@@ -0,0 +1,496 @@
+
+          Make sure you have registered your SSH public key in your GitHub account,
+          via https://github.com/settings/keys.
+
+Before you can open pull requests, you will need to **fork** the GitHub repository
+you are targeting, and create a local **working copy** of that repository. This only
+needs to be done *once* for every of the EasyBuild repositories.
+
+.. note:: For the purpose of this guide, we will be using the ``easybuild-framework`` repository.
+
+          However, the instructions are analogous for the other EasyBuild repositories.
+
+.. _contributing_back_pull_request_setup_fork_repo:
+
+Forking the repository
+++++++++++++++++++++++
+
+First, create your own *fork* of the repository in your GitHub account using the
+'``Fork``' button on the top right at https://github.com/hpcugent/easybuild-framework.
+
+This will basically create a 'copy' of the ``easybuild-framework`` repository
+owned by the ``hpcugent`` GitHub organisation to your own personal GitHub account.
+
+You will have to use this fork as a staging area for your work,
+to prepare your contributions before creating the actual pull requests.
+
+.. _contributing_back_pull_request_setup_working_copy:
+
+Creating a working copy
++++++++++++++++++++++++
+
+In addition to forking the repository on GitHub, you also need to create
+a local *working copy* of the repository. This is basically a local checkout
+of the repository where you can track the changes you are making.
+
+First, *clone* the repository:
+
+.. code:: shell
+
+  git clone git@github.com:hpcugent/easybuild-framework.git
+
+If that worked as expected, you should have a new directory named ``easybuild-framework``.
+Move into the ``easybuild-framework`` directory:
+
+.. code:: shell
+
+  cd easybuild-framework
+
+Then, add your own fork as a '*remote*' repository:
+
+.. code:: shell
+
+  git remote add myfork git@github.com:EXAMPLE/easybuild-framework.git
+
+.. note:: **Do not forget to replace** '``EXAMPLE``' **with your GitHub account name.**
+
+          '``myfork``' is just a name that you give to the remote for your
+          fork of the repository on GitHub; you can modify this to your liking
+          if desired (but do take that into account for the further instructions if you do so).
+
+          For the central repository, a default remote named ``origin`` should
+          have been created via ``git clone`` (see the output of ``git remote -v``).
+
+Finally, it is recommended to also check out the ``develop`` branch,
+which will be used as the base for your own branches:
+
+.. code:: shell
+
+  git checkout -b develop origin/develop
+
+With this in place, you are all set to open pull requests for your contributions.
+
+.. _contributing_back_creating_pull_requests:
+
+Opening a new pull request
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note:: This section describes the manual procedure to open a new pull request.
+
+          Please consider using ``eb --new-pr`` instead, see :ref:`github_new_pr`.
```

Comment:
I'll do this in the `--new-pr` documentation itself, since it's less likely we'll forget to update that once ``easybuild-framework`` and ``easybuild-easyblocks`` are also supported in ``--new-pr`` and ``--update-pr``